### ISSUE TITLE
[BITV] 9.2.5.3/4.2 - Modal dialog "Create new album": The visible label "Create album" does not appear in the accessible name of the button in the same form as it is overridden by the use of aria-label="Create the album". In this case, it cannot be guaranteed that the button will be activated through voice input. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/photos#1675]
- [ ] Fix
- [ ] Backport

![0e4660212b80e42b753b1adcc08a8370](https://user-images.githubusercontent.com/1315170/222545640-f88a4f6e-f352-42ac-964a-c8f074ae61df.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-c6347c127a-v4-n2

</details>
